docker/registry/selenium/gradle.properties

```diff
@@ -1,4 +1,3 @@
 io.deephaven.project.ProjectType=DOCKER_REGISTRY
-deephaven.registry.imageName=selenium/standalone-firefox:4.16.1-20231219
-deephaven.registry.imageId=selenium/standalone-firefox@sha256:a405fe92b3ce5d7eb31a07e1f99be3d628fdc0e5bdc81febd8dc11786edef024
-deephaven.registry.platform=linux/amd64
```

Comment:
I questioned Colin about the removal of this; it was accidental, but in this case, actually _correct_.

This updated selenium image is multiplatform, which can be verified with `docker buildx imagetools inspect selenium/standalone-firefox:4.26.0-20241101` (as compared to `docker buildx imagetools inspect selenium/standalone-firefox:4.16.1-20231219`).